src/Evolution/Systems/Cce/AnalyticSolutions/LinearizedBondiSachs.cpp

```diff
@@ -461,11 +461,11 @@ void LinearizedBondiSachs::dt_spherical_metric(
   get<0, 1>(*dt_spherical_metric) = 0.0;
 
   const ComplexDataVector dt_r_squared_jbar_u =
-      conj(dt_bondi_j.data()) * square(extraction_radius_) * bondi_u.data() +
-      conj(bondi_j.data()) * square(extraction_radius_) * dt_bondi_u.data();
+      square(extraction_radius_) * (conj(dt_bondi_j.data()) * bondi_u.data() +
+                                    conj(bondi_j.data()) * dt_bondi_u.data());
   const ComplexDataVector dt_r_squared_k_ubar =
-      dt_bondi_k.data() * square(extraction_radius_) * conj(bondi_u.data()) +
-      bondi_k.data() * square(extraction_radius_) * conj(dt_bondi_u.data());
+      square(extraction_radius_) * (dt_bondi_k.data() * conj(bondi_u.data()) +
+                                    bondi_k.data() * conj(dt_bondi_u.data()));
   get<0, 2>(*dt_spherical_metric) =
       real(dt_r_squared_jbar_u + dt_r_squared_k_ubar);
   get<0, 3>(*dt_spherical_metric) =
```

Comment:
check the minus sign with respect to the doc and factor out the minus sign of (2,3)